sharpy/aero/utils/mapping.py

```diff
@@ -69,10 +70,11 @@ def aero2struct_force_mapping(aero_forces,
                 cbg = np.dot(cab.T, cag)
 
                 for i_m in range(n_m):
-                    chi_g = zeta[i_surf][:, i_m, i_n] - np.dot(cag.T, pos_def[i_global_node, :])
                     struct_forces[i_global_node, 0:3] += np.dot(cbg, aero_forces[i_surf][0:3, i_m, i_n])
                     struct_forces[i_global_node, 3:6] += np.dot(cbg, aero_forces[i_surf][3:6, i_m, i_n])
-                    struct_forces[i_global_node, 3:6] += np.dot(cbg, algebra.cross3(chi_g, aero_forces[i_surf][0:3, i_m, i_n]))
+                    if not skip_moments_generated_by_forces:
```

Comment:
sorry I'm not very familiar with this bit of the code - but why is this extra toggle for moments necessary?

Reply:
Somehow this moments introduce high unsymmetric forces in the fuselage beam. I could not figure the reason yet. Since we have only looked at rather stiff fuselages for now, the moment around a particular node should not change much. The (global) neutral point shift is mostly introduced by the longitudinal force distribution too. So, I basically neglect these local moments for the fuselage nodes. I added some comments in the code to have it better documented.